### PR TITLE
Updated shipments for PT, AR and MX

### PR DESCRIPTION
**Pull request by lcelso**
## Description
The shipments pages have been updated for both ES (AR and MX) and PT (BR) languages.
That request can be verified here.

Fixes<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [x] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 1]
```diff
@@ -4,11 +4,1048 @@ sites_supported:
 indexable: false
 ---
```
[the remark this answers is not in the document]
Comment (juones) [reply guides/payments/web-payment-checkout/latest/shipments.en.md:5]: @lcelso

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 2]
```diff
@@ -4,11 +4,1048 @@ sites_supported:
...
+```
+```java
+===
+ Respect the dimensions format, in centimeters and grams as appropriate: heightx width x length, weight. 
+===
+Preference preference = new Preference();
+
+Shipments shipments = new Shipments();
+
+// No es obligatorio configurar la propiedad AddressReceiver
```
[the remark this answers is not in the document]
Comment (juones) [reply guides/payments/web-payment-checkout/latest/shipments.en.md:55]: @lcelso

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 3]
```diff
@@ -4,11 +4,1048 @@ sites_supported:
...
+preference.shipment = shipment
+
+```
+```csharp
+===
+ Respect the dimensions format, in centimeters and grams as appropriate: heightx width x length, weight. 
+===
+Preference preference = new Preference();
+
+// No es obligatorio configurar la propiedad ReceiverAddress
```
[the remark this answers is not in the document]
Comment (juones) [reply guides/payments/web-payment-checkout/latest/shipments.en.md:113]: @lcelso

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 4]
```diff
@@ -4,11 +4,1048 @@ sites_supported:
...
+preference.Shipments = shipments;
+```
+]]]
+------------
+----[mlm]----
+
+The shipping cost will be debited from the seller’s account when the payment is received. 
+You can offer different shipping methods by changing the ID. Check the available <a href="https://api.mercadolibre.com/shipping_methods/search?site_id=MLA&shipping_mode=me2&allow_free_shipping=true" target="_blank">id shipping methods</a> to know which ones to add.
+
+For example, in the following code is added the `ID 509247` which refers to a normal home delivery of OCA and the `ID 509245` for normal home delivery of Adreani.
```
[the remark this answers is not in the document]
Comment (juones) [reply guides/payments/web-payment-checkout/latest/shipments.en.md:226]: @lcelso

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 5]
```diff
@@ -4,11 +4,1048 @@ sites_supported:
...
+preference.Shipments = shipments;
+```
+]]]
+------------
+----[mlb]----
+
+The shipping cost will be debited from the seller’s account when the payment is received. 
+You can offer different shipping methods by changing the ID. Check the available <a href="https://api.mercadolibre.com/shipping_methods/search?site_id=MLA&shipping_mode=me2&allow_free_shipping=true" target="_blank">id shipping methods</a> to know which ones to add.
+
+For example, in the following code is added the `ID 505345` which refers to a normal home delivery of OCA and the `ID 100009` for normal home delivery of Adreani.
```
[the remark this answers is not in the document]
Comment (juones) [reply guides/payments/web-payment-checkout/latest/shipments.en.md:305]: @lcelso

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 6]
```diff
@@ -4,11 +4,1048 @@ sites_supported:
...
+shipments.FreeMethods = new List<int> { 505345, 100009 };
+//...
+preference.Shipments = shipments;
+```
+]]]
+------------
+
+#### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Shipping costs simulator
+
+You can simulate costs from the shipping calculator. To do this, you must replace the `sellerId`, `shippingMethod Id`, `price`, `zipCode`, `height`, `anwidthcho`, `length`, `weight` values of your package.
```
[the remark this answers is not in the document]
Comment (juones) [reply guides/payments/web-payment-checkout/latest/shipments.en.md:382]: @lcelso

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 7]
```diff
@@ -4,11 +4,1048 @@ sites_supported:
...
+
+> https://api.mercadolibre.com/users/_sellerId_/shipping_options?free_method=_shippingMethodId_&item_price=_price_&zip_code=_zipCode_&dimensions=_alto_x_ancho_x_largo_,_peso
```
[the remark this answers is not in the document]
Comment (juones) [reply guides/payments/web-payment-checkout/latest/shipments.en.md:384]: @lcelso

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 8]
```diff
@@ -4,11 +4,1048 @@ sites_supported:
...
+//...
+shipments.LocalPickUp = true;
+//...
+preference.Shipments = shipments;
+```
+]]]
+
+**Done! Mercado Envíos is already integrated.** 
+
+Once you receive an order, you only need to <a href="https://www.mercadopago[FAKER][URL][DOMAIN]/ayuda/_1603" target="_blank">prepare and ship the package</a>.
```
[the remark this answers is not in the document]
Comment (juones) [reply guides/payments/web-payment-checkout/latest/shipments.en.md:455]: @lcelso

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 9]
```diff
@@ -4,11 +4,1048 @@ sites_supported:
...
+
+> NOTE
+>
+> Label Management
+>
+> When an order is made, you will receive an email with a button to print the label you must attach to the package. You will also be able to view the <a href="https://www.mercadopago[FAKER][URL][DOMAIN]/activities?type=facet_type_collection&status=facet_shipping_me_all" target="_blank">payments to be printed</a> from the Mercado Pago account that received the order.
```
[the remark this answers is not in the document]
Comment (juones) [reply guides/payments/web-payment-checkout/latest/shipments.en.md:461]: @lcelso

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 10]
```diff
@@ -4,11 +4,1048 @@ sites_supported:
...
+#### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Shipping costs simulator
+
+You can simulate costs from the shipping calculator. To do this, you must replace the `sellerId`, `shippingMethod Id`, `price`, `zipCode`, `height`, `anwidthcho`, `length`, `weight` values of your package.
+
+> https://api.mercadolibre.com/users/_sellerId_/shipping_options?free_method=_shippingMethodId_&item_price=_price_&zip_code=_zipCode_&dimensions=_alto_x_ancho_x_largo_,_peso
+
+[[[
+```curl
+
+curl --location --request GET 'http://api.mercadolibre.com/users/179504451/shipping_options?free_method=182&item_price=718&zip_code=74474322&dimensions=2x11x16,88'
```
[the remark this answers is not in the document]
Comment (juones) [reply guides/payments/web-payment-checkout/latest/shipments.en.md:389]: @lcelso

File: guides/payments/web-payment-checkout/latest/shipments.en.md [hunk 11]
```diff
@@ -1,14 +1,1048 @@
...
+#### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Shipping costs simulator
+
+You can simulate costs from the shipping calculator. To do this, you must replace the `sellerId`, `shippingMethod Id`, `price`, `zipCode`, `height`, `width`, `length`, `weight` values of your package
+
+> https://api.mercadolibre.com/users/_sellerId_/shipping_options?free_method=_shippingMethodId_&item_price=_price_&zip_code=_zipCode_&dimensions=_height_x_width_x_length_,_weight
+
+[[[
+```curl
+
+curl --location --request GET 'http://api.mercadolibre.com/users/179504451/shipping_options?free_method=182&item_price=718&zip_code=[FAKER][SITE][ID]&dimensions=2x11x16,88'
```
Comment (pamalv7) [inline guides/payments/web-payment-checkout/latest/shipments.en.md:389]:
```suggestion
curl --location --request GET 'http://api.mercadolibre.com/users/179504451/shipping_options?free_method=182&item_price=718&zip_code=[FAKER][ADDRESS][ZIP]&dimensions=2x11x16,88'
```

**Pull request merged**